state-chain/runtime/src/chainflip/solana_elections.rs

```diff
@@ -838,3 +893,100 @@ impl ElectionEgressWitnesser for SolanaEgressWitnessingTrigger {
 		)
 	}
 }
+
+#[derive(
+	Clone, PartialEq, Eq, Debug, Serialize, Deserialize, TypeInfo, Encode, Decode, PartialOrd, Ord,
+)]
+pub struct SolanaVaultSwapDetails {
+	pub from: SolAsset,
+	pub to: Asset,
+	pub deposit_amount: SolAmount,
+	pub destination_address: EncodedAddress,
+	pub deposit_metadata: Option<CcmDepositMetadata>,
+	// TODO: These two will potentially be a TransactionId type
```

Comment:
which two? (better to explicitly name in these cases in case code gets moved too, more obvious if it's a stale comment)
```suggestion
	// TODO: These two will potentially be a TransactionId type
```

Reply:
done